lib/gds_api/performance_platform/data_out.rb

```diff
@@ -43,6 +43,119 @@ class DataOut < GdsApi::Base
       def service_feedback(transaction_page_slug)
         get_json("#{endpoint}/data/#{transaction_page_slug}/customer-satisfaction")
       end
+
+      # Fetches data from the performance platform for a given page slug
```

Comment:
It's great to have doc comments for these methods, but they've all got the same description, can we tweak them from `fetches data` to `fetches <whatever> data` so they're more useful? We're also missing the `@param is_multipart` for those methods that take that param.